### ISSUE TITLE
Including parentheses in image alt text breaks image upload

### ISSUE THREAD

**Issue by agc93** [TryGhost/Ghost#8006]
### Issue Summary

Uploading images fails when the alt text includes parentheses

### Steps to Reproduce

1. Create new post
1. Add image markdown to the post, but with parentheses in the alt text (e.g. `![image (alt)]()`)
1. Attempt to upload an image by clicking the placeholder in the preview
1. Image is uploaded to the server and the relative path is inserted into the parentheses **in the alt text**, not the actual image source.

I understand that including parentheses in the alt text is probably pushing Markdown pretty hard, but I haven't seen it specifically disallowed anywhere.

### Technical details:

* Ghost Version: 0.11.0
* Node Version: v4.5.0 (?)
* Browser/OS: Edge (Client) + Docker image (server)
* Database: MariaDB

**Comment by kevinansfield** [comment]
Hey @agc93, thanks for raising this issue! FYI, for anyone looking to work on a fix, I believe the problem lies in our [image manager util](https://github.com/TryGhost/Ghost-Admin/blob/lts/app/utils/ed-image-manager.js). The regex at the top probably needs to be updated to better match the alt text vs the src when the alt text contains parens.

**Comment by nglgzz** [comment]
Hey @kevinansfield, I think I managed to solve this. Are there some tests I should run before creating a PR?

**Comment by kevinansfield** [comment]
Hey @nglgzz, yes tests can be run by switching to the client dir (`cd core/client`) and then running `ember test -s` - that will launch both Chrome and Firefox and keep the browser open so changing a file will re-run the tests, if you only want to test in one browser (our CI system tests both anyway) then you can run `ember test -s --launch=chrome`.

**Comment by kevinansfield** [comment]
Closed via https://github.com/TryGhost/Ghost-Admin/pull/741
